L/librdkafka/build_tarballs.jl

```diff
@@ -3,12 +3,12 @@
 using BinaryBuilder, Pkg
 
 name = "librdkafka"
-version = v"1.8.2"
+version = v"2.0.2"
 
 # Collection of sources required to complete build
 sources = [
-    ArchiveSource("https://github.com/edenhill/librdkafka/archive/v$(version).tar.gz",
-                  "6a747d293a7a4613bd2897e28e8791476fbe1ae7361f2530a876e0fd483482a6"),
+    ArchiveSource("https://github.com/confluentinc/librdkafka/archive/refs/tags/v$(version).tar.gz",
```

Comment:
In case it wasn't clear, this source isn't acceptable, as explained by [the error message](https://buildkite.com/julialang/yggdrasil/builds/1504#0186ed48-ce23-414c-aea9-2533246953f4/538-545).

Reply:
Thanks, fixed.